src/main/java/main/EmployeesProcessor.java

```diff
@@ -0,0 +1,412 @@
+                break;
+            }
+        }
+        return distinctAddressesOfEmployees;
+    }
+
+    /**
+     * TASK #9
+     * Витягнути колекцію імен працівників так, щоб нова колекція включала в себе 5 імен
+     */
+    public List<String> getFiveEmployeesName(List<Employee> employeeList, ExecutionType executionType){
+        List<String> fiveEmployeesName = null;
+        switch (executionType){
+            case FOREACH:{
+                fiveEmployeesName = new ArrayList<>();
+                for (Employee e:employeeList){
+                    if (!(fiveEmployeesName.size() == 5)) {
+                        fiveEmployeesName.add(e.getFirstName());
+                    }
+                }
+                break;
+            }
+            case ITERATOR:{
+                fiveEmployeesName = new ArrayList<>();
+                Iterator<Employee> employeeIterator = employeeList.iterator();
+                while (employeeIterator.hasNext()){
+                    String name = employeeIterator.next().getFirstName();
+                    if (!(fiveEmployeesName.size() == 5)) {
+                        fiveEmployeesName.add(name);
+                    }
+                }
+                break;
+            }
+            case STREAM:{
+                fiveEmployeesName = employeeList.stream()
+                        .map(s->s.getFirstName())
+                        .limit(5)
+                        .collect(Collectors.toList());
+            }
+            break;
+        }
+        return fiveEmployeesName;
+    }
+
+    /**
+     * TASK #10
+     * Порахувати суму віку всіх працівників
+     */
+    public Integer getSumOfAge(List<Employee> employeeList, ExecutionType executionType){
+        Integer sumOfAge = 0;
+        switch (executionType){
+            case FOREACH:{
+                for (Employee e:employeeList){
+                    sumOfAge += e.getAge();
+                }
+                break;
+            }
+            case ITERATOR:{
+                Iterator<Employee> employeeIterator = employeeList.iterator();
+                while (employeeIterator.hasNext()){
+                    sumOfAge += employeeIterator.next().getAge();
+                }
+                break;
+            }
+            case STREAM:{
+                sumOfAge = employeeList.stream()
+                        .map(s->s.getAge())
+                        .reduce(0,Integer::sum);
+            }
+            break;
+        }
+        return sumOfAge;
+    }
+
+    /**
+     * TASK #11
+     * Перевірити, чи всі працівники старше 18ти
+     */
+    public boolean checkIfAllEmployeesOldestEighteen(List<Employee> employeeList, ExecutionType executionType){
+        boolean allEmployeesOldest = false;
+        switch (executionType){
+            case FOREACH:{
+                int countAge = 0;
+                for (Employee e:employeeList){
+                    if (e.getAge()>18){
+                        countAge++;
```

Comment:
The task is to find if All of workers are older than 18. What is mean that if even one worker is 18 or less - than ALL workes is Not 18, isn't it? So, again, if we will have 10000000000000000000000000000 workers, should we itarate through everyone if we need to find just a one who is 18 or less?:)